Notify/Notify/Notify/ViewModels/FriendRequestPageViewModel.cs

```diff
@@ -83,6 +107,17 @@ private async void PopulateUsersList()
             UsersList = JsonConvert.DeserializeObject<List<Friend>>(usersListJson);
             UsersSelectionList = UsersList = await AzureHttpClient.Instance.GetNotFriendsUsers();
         }
+        
+        private async void PopulateFriendRequestsList()
+        {
+            //string usersListJson = Preferences.Get(Constants.PREFERENCES_FRIEND_REQUESTS, "");
+            
+            //UsersList = JsonConvert.DeserializeObject<List<Friend>>(usersListJson);
```

Comment:
Do we need this?

Reply:
Not at the moment.
Should be on a different branch. Removing it now.